doc/nrf/external_comp/nrf_cloud.rst

```diff
@@ -34,6 +34,46 @@ You can use the services offered by nRF Cloud in the following scenarios:
 * Device connected to nRF Cloud over REST, interacting using the `nRF Cloud REST API`_.
 * Device connected to a customer cloud service in a suitable manner. The services can be used from the customer cloud service that communicates over REST to the nRF Cloud REST API in a proxy configuration.
 
+Choosing a device ID format: prefix-IMEI, device UUID, or other
+***************************************************************
+
+nRF Cloud supports a variety of device ID formats:
+
+* A prefix plus IMEI
+  This is reserved for Nordic development kits or evaluation platforms when the prefix is ``nrf-``.
+* The device UUID
+  This is a guaranteed unique ID stored in the modem.
+* Compile-time
+  A fixed device ID compiled into the firmware.
+* Run-time
+  A device ID generated by the application in an application-specific way.
+* HW-ID
+  A device ID generated from one of a selection of hardware IDs.
+
+Offical Nordic development kits such as the nRF9160DK have a sticker on the back indicating the device's IMEI and PIN.
+They are preprovisioned with JITP certificates so they can be added easily to nRF Cloud using the device ID ``nrf-IMEI`` and PIN.
+Customers using custom hardware cannot use ``nrf-<IMEI>`` style device ids.
+Names that start with ``nrf-`` and their PIN or HWID are checked against a database of genuine Nordic development kits and Thingy:91s.
+If not found in the database, the device cannot connect.
+PINs/HWIDs are not needed for customers to connect custom devices to nRF Cloud.
+
+The device_credentials_installer.py script supports either method.
```

Comment:
I think we should link to the repo that contains this script when mentioning it here for the first time